test/1_stdlib/StringDiagnostics.swift

```diff
@@ -98,3 +98,9 @@ func testStringCollectionTypes(s: String) {
   acceptsBidirectionalCollection(s.characters)
   acceptsRandomAccessCollection(s.characters) // expected-error{{argument type 'String.CharacterView' does not conform to expected type 'RandomAccessCollection'}}
 }
+
+struct NotLosslessStringConvertible {}
+
+func testStringInitT() {
+  _ = String(NotLosslessStringConvertible()) // expected-error{{'init' has been renamed to 'init(describing:)}}
```

Comment:
This should provide a fixit hint to do the renaming, right?  If so, please add a test for it.  Add {{1-2=}} to the end of the expected-error line and run the test.  The test will fail and tell you what the right thing is.  Thanks!